test/tests/transition-aborted-error_test.js

```diff
@@ -0,0 +1,20 @@
+import { module, test } from './test_helpers';
+import TransitionAbortedError from 'router/transition-aborted-error';
+
+module('transition-aborted-error');
+
+test('correct inheritance and name', function(assert) {
+  var error;
+
+  try {
+    throw new TransitionAbortedError('Message');
+  } catch(e) {
+    error = e;
+  }
+
+  // it would be more correct with TransitionAbortedError, but other libraries may rely on this name
+  assert.equal(error.name, 'TransitionAborted', "TransitionAbortedError has the name 'TransitionAborted'");
```

Comment:
FWIW, Ember itself expects `.name` to be `TransitionAborted`: https://github.com/emberjs/ember.js/blob/aeeddf7c00b42a340494465252481b5414d87009/packages/ember-runtime/lib/ext/rsvp.js#L40-L42